evm_arithmetization/src/generation/mod.rs

```diff
@@ -486,7 +486,11 @@ fn get_all_memory_address_and_values(memory_before: &MemoryState) -> Vec<(Memory
     res
 }
 
-type TablesWithPVsAndFinalMem<F> = ([Vec<PolynomialValues<F>>; NUM_TABLES], PublicValues<F>);
+pub struct TablesWithPVsAndFinalMem<F: RichField> {
```

Comment:
Nit: out of curiosity: why `AndFinalMem`?

Reply:
Renamed it. Thanks for the catch!